content/ipedrazas/drone-helm/index.md

```diff
@@ -0,0 +1,99 @@
+---
+date: 2017-02-08T00:00:00+00:00
+title: Helm
+author: ipedrazas
+tags: [ kubernetes, docker, helm ]
+repo: ipedrazas/drone-helm
+logo: kubernetes.svg
+image: ipedrazas/drone-helm
+---
+
+This plugin allows you to run Kubernetes Helm as part of your pipeline.
+
+```nohighlight
+helm init
+helm upgrade --install RELEASE CHART
+```
+
+The plugin initialises helm first, then it issues the install/upgrade command. There are a few settings that
+map to the equivalent `helm` command.
+
+Basic example:
+
+```yaml
+pipeline:
+    helm_deploy:
+        image: quay.io/ipedrazas/drone-helm
+        skip_tls_verify: true
+        chart: ./charts/my-chart
+        release: ${DRONE_BRANCH}
+        values: secret.password=${SECRET_PASSWORD},image.tag=${TAG}
+        prefix: STAGING
+        namespace: development
+```
+
+This block will execute the following command:
+
+```
+helm upgrade --install ${DRONE_BRANCH} ./charts/my-chart --namespace development --set secret.password=${SECRET_PASSWORD},image.tag=${TAG}
+```
+
+The plugin expects two secrets defined to connect to the kubernetes cluster:
+
+```
+drone secret add --image=quay.io/ipedrazas/drone-helm \
+        your-user/your-repo STAGING_API_SERVER https://mykubernetesapiserver
+
+drone secret add --image=quay.io/ipedrazas/drone-helm \
+        your-user/your-repo STAGING_KUBERNETES_TOKEN eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJrdWJ...
+```
```

Comment:
I'll have to test this ([drone/drone@1f0261a](https://github.com/drone/drone/commit/1f0261a72a6c73428918f76f35d2e4f3bfaf057b)), but this will make the plugin behave in a very different way.